djangocms_maps/static/djangocms_maps/js/mapbox.js

```diff
@@ -85,7 +85,7 @@ djangocms.Maps = {
         // latitute or longitute have precedence over the address when provided
         // inside the plugin form
         if (data.lat.length && data.lng.length) {
-            var latlng = { lat: parseFloat(data.lat), lng: parseFloat(data.lng) };
+            var latlng = { lat: parseFloat(data.lat.replace(',', '.')), lng: parseFloat(data.lng.replace(',', '.')) };
```

Comment:
Can you maybe put the code below, like so, and see if JavaScript is still happy?
```javascript
var latlng = {
    lat: parseFloat(data.lat.replace(',', '.')),
    lng: parseFloat(data.lng.replace(',', '.'))
 };
```

Reply:
@philippze Would you be willing to make that change? I'd merge the changes afterwards.

Reply:
Hallo @bittner , yes! I should find the time today.